### PR TITLE
zscp: argument checks for hypothesis template and nli indexes (#551)

### PR DESCRIPTION
**Pull request by kylesayrs**
Adds errors that trigger on pipeline creation

### CODE REVIEW

File: src/deepsparse/transformers/pipelines/mnli_text_classification.py [hunk 1]
```diff
@@ -125,6 +125,27 @@ def __init__(
...
+        if (
+            self._config.hypothesis_template is not None
+            and self._config.hypothesis_template.format("sample_label")
+            == self._config.hypothesis_template
+        ):
+            raise ValueError(
+                "The provided hypothesis_template "
+                f"`{self._config.hypothesis_template}` was not able to be formatted. "
+                "Make sure the passed template includes formatting syntax such "
+                "as {} where the label should go."
```
Comment (bfineran) [inline src/deepsparse/transformers/pipelines/mnli_text_classification.py:137]:
```suggestion
                "as '{}' where the label should go."
```

**Comment by bfineran** [comment]
make sure to update description

**Comment by kylesayrs** [comment]
Oops, @bfineran  accidentally committed your suggestion. Opening another PR with one commit

**Comment by kylesayrs** [comment]
https://github.com/neuralmagic/deepsparse/pull/561

**Pull request closed without merging**